api/admin/controller/view.py

```diff
@@ -25,10 +26,12 @@ def __call__(self, collection, book, path=None):
                 redirect_url = flask.request.url
                 if collection:
                     redirect_url = redirect_url.replace(
-                        collection, url_quote_plus(collection)
+                        collection, quote_plus(collection, safe="()")
```

Comment:
The deprecation message for `url_quote_plus` recommended to replace it with a call to `quote_plus`. 

This causes our tests to fail because we assume that this function doesn't quote `()`, so I added in the `safe` parameter to make sure that brackets are not escaped.